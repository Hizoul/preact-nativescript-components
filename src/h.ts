import * as preact from "preact"
import { Preact } from "preact-to-nativescript"

const customH: any = Preact.h

export default customH
