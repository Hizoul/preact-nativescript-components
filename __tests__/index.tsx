import * as renderToJson from "preact-render-to-json"
import Label from "../src/components/label"
import h from "../src/h"

test("Label Render", () => {
  expect(
    renderToJson(<Label text={"my text"} />)
  ).toMatchSnapshot("label render")
})
