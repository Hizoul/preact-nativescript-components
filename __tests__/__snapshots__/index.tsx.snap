// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`label render 1`] = `
<label
  text="my text"
/>
`;
